src/client/static/context.js

```diff
@@ -0,0 +1,39 @@
+Vue.component('context-box', {
+    name: 'context-box',
+    data: function () {
+        return {
+            isCollapsed: false
+        }
+    },
+    template: `
+    <div>
+        <p>
+            <button class="btn btn-primary" v-on:click="isCollapsed = !isCollapsed">
+            Information
+            </button>
+        </p>
+        <div class="card card-body" v-if="isCollapsed">
+        <h4>Welcome to RedditVis!</h4>
+
+        <p><b>RedditVis</b> is a multiview visual analytics system that allows to explore the network of 
+        subreddits relations and analyse the details of linguistic and semantic post properties 
+        between selected subreddits. You can interactively select subreddits of interest that exist 
+        in the network and explore further details. Identify relationships, 
+        patterns and trends within the data.</p>
```

Comment:
In my opinion this would be enough. 
Maybe button could be on the navigation and say "About RedditVis".

Also there is a punctuation mistake (here fixed):
``` further details, identify relationships, patterns and trends within the data.```

Reply:
ex. 
![image](https://user-images.githubusercontent.com/72405859/112959017-4caa5600-9143-11eb-8b08-990de3cc20aa.png)

Reply:
it was first in the navigation bar but i couldnt get it to work like that sadly

Reply:
I will put it in the navbar this afternoon.

Reply:
also, the modal could be used instead of collapsible area. would keep the page more static